pkg/cmd/template/schema_author_test.go

```diff
@@ -493,12 +493,12 @@ Invalid schema - @schema/default is wrong type
 
 schema.yml:
     |
+  3 | #@schema/default 1
   4 | foo: a string
     |
 
-    = found: integer
+    = found: integer (by schema.yml:3)
```

Comment:
```suggestion
    = found: integer (at schema.yml:3)
```

🤷🏻 

My case for "at" vs "by" is that "by" makes more sense for the "expected" situation... "string is expected _by_ <file:pos>" vs. "an integer was found _at_ <file:pos>"